### PR TITLE
Search /usr/share for rpmdb to fix scan on ostree-managed images

### PR DESCRIPTION
**Pull request by fpytloun**
fixes: https://github.com/anchore/syft/issues/1755

### CODE REVIEW

File: syft/pkg/rpm_metadata.go [hunk 1]
```diff
@@ -11,7 +11,7 @@ import (
 // Packages is the legacy Berkely db based format
 // Packages.db is the "ndb" format used in SUSE
 // rpmdb.sqlite is the sqlite format used in fedora + derivates
-const RpmDBGlob = "**/var/lib/rpm/{Packages,Packages.db,rpmdb.sqlite}"
+const RpmDBGlob = "**/{var/lib,usr/share}/rpm/{Packages,Packages.db,rpmdb.sqlite}"
```
Comment (kzantow) [inline syft/pkg/rpm_metadata.go:14]: Would you be able to add a test that has a simple rpmdb in this alternate location?

**Comment by kzantow** [comment]
@fpytloun you'll also need to [sign-off your commits](https://github.com/anchore/syft/blob/main/CONTRIBUTING.md#sign-off-your-work)

File: syft/pkg/rpm_metadata.go (hunk 1 above)
Comment (ivanstanev) [reply syft/pkg/rpm_metadata.go:14]: Side note: if you'd like to also support OpenSUSE, you might also need `/var/lib/sysimage/` like in https://github.com/snyk/snyk-docker-plugin/commit/27958de3366ab6fd1b7f7faad64cce8f0dce5350

**Comment by wagoodman** [comment]
I added a cataloger glob test to make certain the cataloger is selecting the right paths, so the PR should be good to go once you sign off your commit @fpytloun . 👍

**Comment by fpytloun** [comment]
Thank you @wagoodman 👍 

Signed commit, should be ready now 🙂

**Comment by fpytloun** [comment]
Sorry, to fix DCO I had to rebase and squash your commit into mine with `co-authored-by`, @wagoodman

**Review by wagoodman — APPROVED** [review]
Thanks for the contribution @fpytloun 🙌

**Pull request merged**